app/dashboard/static/js/app/tables/common.js

```diff
@@ -276,7 +276,7 @@ define([
                 rendered = node.outerHTML;
             } else {
                 created = new Date(date.$date);
-                rendered = created.toCustomISODate();
+                rendered = created.toCustomISODateTime();
```

Comment:
@gctucker I updated this method to consider date time, local it was fine..I also looked for side-effects in builds and boot views and seems to be ok! Let's check on staging, because it's used in many places.

Reply:
But, the other tables didn't have an issue.  So why fix the common.js file which is used by other tables?

Reply:
because all tables are using `common.js`, most part of the methods from boot, build, etc. If I am not wrong, every time `renderDate` is called actually it is calling `renderDate` in `common.js` file and now all files using common are calling `.toCustomISODateTime()`.

Reply:
Yes, but before you made this change, the other tables were working fine.  So I don't think this needs to be changed.

Reply:
ops, yes! You are right.